test/controllers/update/check.your.answers.controller.spec.ts

```diff
@@ -0,0 +1,98 @@
+jest.mock("ioredis");
+jest.mock('../../../src/service/transaction.service');
+jest.mock('../../../src/service/overseas.entities.service');
+jest.mock('../../../src/service/payment.service');
+jest.mock('../../../src/middleware/authentication.middleware');
+jest.mock('../../../src/middleware/service.availability.middleware');
+jest.mock('../../../src/utils/application.data');
+
+import { NextFunction, Request, Response } from "express";
+import { beforeEach, expect, jest, test, describe } from "@jest/globals";
+import request from "supertest";
+import { serviceAvailabilityMiddleware } from "../../../src/middleware/service.availability.middleware";
+
+import {
+  UPDATE_CHECK_YOUR_ANSWERS_PAGE,
+  UPDATE_CHECK_YOUR_ANSWERS_URL
+} from "../../../src/config";
+import app from "../../../src/app";
+import {
+  PAGE_TITLE_ERROR,
+  SERVICE_UNAVAILABLE,
+  UPDATE_CHECK_YOUR_ANSWERS_PAGE_TITLE
+} from "../../__mocks__/text.mock";
+import {
+  ERROR,
+  OVERSEAS_ENTITY_ID,
+  APPLICATION_DATA_MOCK,
+  TRANSACTION_CLOSED_RESPONSE,
+  PAYMENT_LINK_JOURNEY,
+  TRANSACTION_ID
+} from "../../__mocks__/session.mock";
+
+import { authentication } from "../../../src/middleware/authentication.middleware";
+import { postTransaction, closeTransaction } from "../../../src/service/transaction.service";
+import { createOverseasEntity } from "../../../src/service/overseas.entities.service";
+import { startPaymentsSession } from "../../../src/service/payment.service";
+import { getApplicationData } from "../../../src/utils/application.data";
+
+const mockGetApplicationData = getApplicationData as jest.Mock;
+const mockAuthenticationMiddleware = authentication as jest.Mock;
+mockAuthenticationMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockServiceAvailabilityMiddleware = serviceAvailabilityMiddleware as jest.Mock;
+mockServiceAvailabilityMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockTransactionService = postTransaction as jest.Mock;
+mockTransactionService.mockReturnValue( TRANSACTION_ID );
+
+const mockOverseasEntity = createOverseasEntity as jest.Mock;
+mockOverseasEntity.mockReturnValue( OVERSEAS_ENTITY_ID );
+
+const mockCloseTransaction = closeTransaction as jest.Mock;
+mockCloseTransaction.mockReturnValue( TRANSACTION_CLOSED_RESPONSE );
+
+const mockPaymentsSession = startPaymentsSession as jest.Mock;
+mockPaymentsSession.mockReturnValue( "CONFIRMATION_URL" );
+
+describe("CHECK YOUR ANSWERS controller", () => {
+
```

Comment:
for consistency better do not leave empty line after '{' or before `}` in one please and do not add in other

```
describe("GET tests", () => {
-> added new line
    test(`renders the ${UPDATE_CHECK_YOUR_ANSWERS_PAGE} page`, async () => {
    -> not added new line
```

Reply:
fixed